src/create-model.jl

```diff
@@ -604,6 +604,13 @@ function create_model(
         Ai = Dict(y => filter_graph(graph, A, true, :investable, y) for y in Y)
         Fi = Dict(y => filter_graph(graph, F, true, :investable, y) for y in Y)
 
+        Yi = Dict(
+            a => [y for y in Y if a in Ai[y]] for a in A if any(graph[a].investable[y] for y in Y)
+        )
```

Comment:
I found this a bit complicated, but I don't have a better solution. Maybe add a comment to indicate that this is inverting `Ai`?

Reply:
Thank you! Exactly, you get it although it look a bit ugly, this is inverting `Ai`, which probably means it is correct. I'll add a comment.